packages/theia-integration/src/browser/diagram/glsp-diagram-widget.ts

```diff
@@ -49,6 +52,11 @@ export class GLSPDiagramWidget extends DiagramWidget implements SaveableSource {
     protected options: DiagramWidgetOptions & GLSPWidgetOptions;
     protected requestModelOptions: Args;
 
+    protected readonly onWidgetClosedEmitter = new Emitter<undefined>();
```

Comment:
`undefined` seems like a antipattern. Wouldn't `void` be better for this case?

Reply:
I removed the EventEmitter (see comment below) now.